core/domain/stats_services.py

```diff
@@ -212,6 +213,165 @@ def handle_stats_creation_for_new_exp_version(
     create_stats_model(exploration_stats)
 
 
+def create_exp_issues_for_new_exploration(exp_id, exp_version):
+    """Creates the ExplorationIssuesModel instance for the exploration.
+
+    Args:
+        exp_id: str. ID of the exploration.
+        exp_version: int. Version of the exploration.
+    """
+    stats_models.ExplorationIssuesModel.create(exp_id, exp_version, [])
+
+
+def update_exp_issues_for_new_exp_version(exploration, change_list):
+    """Retrieves the ExplorationIssuesModel for the old exp_version and makes
+    any required changes to the structure of the model.
+
+    Args:
+        exploration: Exploration. Domain object for the exploration.
+        change_list: list(dict). A list of changes introduced in this commit.
```

Comment:
Would it be possible to use a list of domain objects instead?

Reply:
@seanlip  Well, the ExpVersionsDiff domain object mapping uses the change_list dict as it is here. We don't use this dict anyway other than passing it as a constructor to the ExpVersionsDiff. Maybe make this at the outset and pass in the ExpVersionsDiff object?

Reply:
Yep, I think passing around domain objects would be better.

Reply:
Done.